NDVItimeseries.R

```diff
@@ -0,0 +1,527 @@
+    axis.title.y = element_text(angle = 0, vjust = 0.5),      # Rotate y axis so don't have to crank head
+    legend.position="bottom"
+  ) 
+
+# Don't show scientific notation
+options(scipen = 999)
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Load Libraries and Options -----
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+library(tidyverse)
+library(haven) # for read_dta
+library(tidylog) # for output on operations/changes with select, drop, merges, etc.
+library(rio)
+library(janitor)
+library(sf) # for st_read
+
+library(openxlsx)
+library(readxl)
+library(moderndive) 
+library(readr)
+library(tidyr)
+library(lubridate) # for dates
+
+library(reshape2)
+library(ggplot2) # included in tidyverse
+library(ggthemes)
+library(stringr)
+library(dplyr)
+library(openxlsx)
+library(readr)
+library(sf)
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Load Data -----
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+#admin 2
+
+NDVI1admin2 <-read.csv("C:/Users/riley/OneDrive/Desktop/My classes/Economics/New folder/DSPG 2022/DSPG_Sahel/Sahel Data/NDVIMean_ts_niger (1981-2000).csv")
+
+NDVI2admin2<-read.csv("C:/Users/riley/OneDrive/Desktop/My classes/Economics/New folder/DSPG 2022/DSPG_Sahel/Sahel Data/NDVIMean_ts_niger (2001-2022).csv")
+
+#admin 3
+
+NDVI1admin3 <-read.csv("C:/Users/riley/OneDrive/Desktop/My classes/Economics/New folder/DSPG 2022/DSPG_Sahel/Sahel Data/l3NDVIMean_ts_niger (1981-2000).csv")
+
+NDVI2admin3 <- read.csv("C:/Users/riley/OneDrive/Desktop/My classes/Economics/New folder/DSPG 2022/DSPG_Sahel/Sahel Data/l3NDVIMean_ts_niger (2001-2022).csv")
+
+#merge datasets
+NDVIadmin2 <- dplyr::left_join(NDVI1admin2, NDVI2admin2)
+NDVIadmin3 <- dplyr::left_join(NDVI1admin3, NDVI2admin3)
+
+# geospatial
+nigerMapAdmin2 <- 
+  st_read("C:/Users/riley/OneDrive/Desktop/My classes/Economics/New folder/DSPG 2022/DSPG_Sahel/Sahel Data/Level 2 (1)/wb_niger_admin2_shapefile/niger_admin2.shp") 
+
+
+nigerMapAdmin3 <- 
+  st_read("C:/Users/riley/OneDrive/Desktop/My classes/Economics/New folder/DSPG 2022/DSPG_Sahel/Sahel Data/Level 3 (1)/NER_adm03_feb2018.shp")
+
+
+# Don't show scientific notation
+options(scipen = 999)
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Clean titles and data -----
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+for ( col in 1:ncol(NDVIadmin2)){
+  colnames(NDVIadmin2)[col] <- sub("_NDVI", "", colnames(NDVIadmin2)[col])
+}
+for ( col in 1:ncol(NDVIadmin2)){
+  colnames(NDVIadmin2)[col] <- sub("X", "", colnames(NDVIadmin2)[col])
+}
+
+for ( col in 1:ncol(NDVIadmin3)){
+  colnames(NDVIadmin3)[col] <- sub("_NDVI", "", colnames(NDVIadmin3)[col])
+}
+for ( col in 1:ncol(NDVIadmin3)){
+  colnames(NDVIadmin3)[col] <- sub("X", "", colnames(NDVIadmin3)[col])
+}
+head(NDVIadmin3)
+
+
+# Convert data from Wide to Long ---- 
```

Comment:
add section divider above and below this section title %%%%%%%%%

Reply:
In this section you are also adding 'date' column. Put this code into next section under 'Convert Column Names to Dates and create new columns -----'